app/models/task.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :integer          not null, primary key
+#  name        :string
+#  description :text
+#  state       :integer          default(-1)
+#  project_id  :integer
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_tasks_on_name  (name) UNIQUE
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  validates :name, presence: true
+  validates :description, presence: true
```

Comment:
Since a task can't really exist with any meaning outside of a project it would probably make sense to add:
`validates :project, presence: true`